src/renderer/react/app/register/Register.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { setKey } from 'services/state';
+
+import RegisterPresentation from './RegisterPresentation';
+
+export const RegisterContainer = Presentation => class Register extends React.PureComponent {
```

Comment:
HOCs shouldn't be Pure. It should leave the re-render logic to the presentation.

Reply:
Resolved